types.go

```diff
@@ -382,3 +382,29 @@ type Element struct {
 	// Comment is an optional comment for the element
 	Comment *string
 }
+
+type FlowtableHookPriority string
+
+const (
+	// FilterHookPriority stands for 0
+	FilterHookPriority FlowtableHookPriority = "filter"
+)
+
+// Flowtable represents an nftables flowtable.
+// https://wiki.nftables.org/wiki-nftables/index.php/Flowtables
+type Flowtable struct {
+	// Name is the name of the flowtable.
+	Name string
+
+	// The Priority can be a signed integer or FilterHookPriority which stands for 0.
+	// Addition and subtraction can be used to set relative priority, e.g. filter + 5 equals to 5.
+	Priority *FlowtableHookPriority
+
+	// The Devices are specified as iifname(s) of the input interface(s) of the traffic
+	// that should be offloaded. Devices are required for both traffic directions.
```

Comment:
I don't understand what that means ("required for both traffic directions"). Does that mean you need to include both the external NIC and the pod veth?

Reply:
copied directly from the netfilter page, and yes, I think it means that

Reply:
I will omit it as it is not clear what it means, I don't even know if you can offload only one part of the traffic and then you have asymetric behavior